profiles/kentik_snmp/cisco/cisco-firepower.yml

```diff
@@ -32,26 +28,105 @@ metrics:
         name: cpmCPUTotal1minRev
         tag: CPU
```

Comment:
Just wondering, if we are moving the memory measurement to the firepower mib, shouldn't we also take CPU from cfprSmMonitorCpuTotalLoadAvg1min 1.3.6.1.4.1.9.9.826.1.71.20.1.6?

Reply:
Makes sense to me then, thanks for checking.